packages/api/src/routers/svc/reminders.ts

```diff
@@ -67,7 +68,7 @@ export function remindersServiceRouter() {
 
       if (!reminders) {
         console.log('reminders not found', userId, scheduleTime)
-        res.status(404).send('Not Found')
+        res.status(404).send('Reminders Not Found')
```

Comment:
If there are no reminders, should we just return `200` so the task isn't retried?

Reply:
Good catch!